test_scenarios/3idx3sh1cm_upgrade.yaml

```diff
@@ -0,0 +1,200 @@
+version: "3.6"
+
+networks:
+  splunknet:
+    driver: bridge
+    attachable: true
+
+volumes:
+  so1-etc:
```

Comment:
you probably would want a named volume for each individual machine, not reusing the same name.  If multiple splunk's get scheduled to the same host, this will break.